ceph-releases/jewel/ubuntu/14.04/demo/Dockerfile

```diff
@@ -7,8 +7,17 @@
 FROM ceph/base:tag-build-master-jewel-ubuntu-14.04
 MAINTAINER Sébastien Han "seb@redhat.com"
 
+# Install s3cmd to support creation of buckets
+RUN ["apt-get", "update", "-y"]
```

Comment:
It's preferred for CMD at least, so I've always used it for RUN and EXEC just for consistency.

https://docs.docker.com/engine/reference/builder/#/cmd